src/components/view/news/news.ts

```diff
@@ -0,0 +1,42 @@
+import IItem from '../../interfaces/IItems';
+import INewsItem from '../../interfaces/INewsItem';
+import './news.css';
+
+class News {
+    public draw(data: []) {
+        const news = (data.length >= 10 ? data.filter((_item: IItem, idx: number) => idx < 10) : data) as [];
```

Comment:
With typecasting as [] you have a type never[]. Suppose it's not what you need here